ament_tools/commands/ament.py

```diff
@@ -30,6 +31,8 @@ def main(sysargs=None):
     # Assign sysargs if not set
     sysargs = sys.argv[1:] if sysargs is None else sysargs
 
+    os.environ["AMENT_TOOOLS"] = "true"
```

Comment:
Spelling: `AMENT_TOOLS`

Nitpick: when updating the line please also use single quotes instead of double quotes.